test/batcher_test.go

```diff
@@ -0,0 +1,366 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+package test
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/hyperledger/fabric-x-orderer/common/tools/armageddon"
+	"github.com/hyperledger/fabric-x-orderer/common/types"
+	"github.com/hyperledger/fabric-x-orderer/testutil"
+	"github.com/hyperledger/fabric-x-orderer/testutil/client"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// Number of shards in the test
+	numOfShards = 4
+	// Number of parties in the test
+	numOfParties = 4
+	// Placeholder for a value that doesn't matter in the test
+	doesntMatter = -1
```

Comment:
please remove this 
it is easier to follow if you just use `-1` instead of this